src/api/app/models/comment.rb

```diff
@@ -14,6 +14,8 @@ class Comment < ApplicationRecord
 
   validate :validate_parent_id
 
+  validates_with CommentLockingValidator
```

Comment:
What is the reason for this to go in `comment.rb` model instead of `comment_lock.rb`?